README.md

```diff
@@ -1 +1,93 @@
-# tic-tac-toe
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Run on Repl.it](https://repl.it/badge/github/edxco/tic-tac-toe)](https://repl.it/github/edxco/tic-tac-toe)
+# Tic-Tac-Toe
+
+> The famous game: Tic-Tac-Toe
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+- Object Oriented Programming
+
+## Live Demo
+
+[Live Demo Link](https://repl.it/github/edxco/tic-tac-toe)
+
+
+## Getting Started
+ You need 2 players to play the game!
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- [Ruby](https://www.ruby-lang.org/en/downloads/) installed on your machine
+
+### Setup
+Clone or download the source code, and run it in your teminal.
+
+### Game instructions
+Start the game by running this code in your teminal `bin/main.rb`.
+
+
+You start the game by entering your names, then the computer will randomly choose one of you to choose his sign. You can only choose "x" or "o".
+After choosing your signs, the first player will choose from 1-9 to put his sign in that index as shown in below image:
+
+![screenshot](https://github.com/edxco/tic-tac-toe/blob/readme_game_instructions/images/modi5.jpg)
+
+Once the first player choose an index, that index will be changed to his sign as shown below:
+
+![screenshot](https://github.com/edxco/tic-tac-toe/blob/readme_game_instructions/images/modi4.jpg)
+
+Then it's other player turn. The second player now should choose his sign in order not to allow the first player to win.
+
+### Win cases
```

Comment:
There are 9 winning cases. In addition to the job done so far, a description of what is a winning case.